tests/middleware_exceptions/middleware.py

```diff
@@ -0,0 +1,6 @@
+from django.http import HttpResponse
```

Comment:
I think adding `from __future__ import unicode_literals` to this file will fix the test failure on Python 3.